Minecraft_BTE_Tips/index.html

```diff
@@ -0,0 +1,140 @@
+  <link rel="stylesheet" href="./main.css">
+  <link rel="icon" href="../Assets/Images/icons/aseanbte_logo.png">
+  <script src="https://kit.fontawesome.com/ce6ee433fd.js" crossorigin="anonymous"></script>
+  <title>ASEAN-BTE| Minecraft_BTE_Tips</title>
+</head>
+
+<body>
+  <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+    <div class="container-fluid">
+      <img src="../Assets/Images/icons/aseanbte_logo.gif" height="60" width="60">
+      <a href="../index.html" class="navbar-brand">&nbsp;ASEAN</a>
+      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent"
+        aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+          <li class="nav-item">
+            <a href="../Projects/projects.html" class="nav-link p-3" style="font-size: large;">
+              <i class="fas fa-tasks"></i>&nbsp;Projects</a>
+          </li>
+          <li class="nav-item">
+            <a href="../Contact_Us/contact.html" class="nav-link p-3" style="font-size: large;">
+              <i class="fa fa-fw fa-envelope"></i>&nbsp;Contact Us</a>
+          </li>
+          <li class="nav-item">
+            <a href="../About_Us/index.html" class="nav-link p-3" style="font-size: large;">
+              <i class="fas fa-atlas"></i>&nbsp;About Us</a>
+          </li>
+          <li class="nav-item">
+            <a href="../Teams/index.html" class="nav-link p-3" style="font-size: large;">
+              <i class="fas fa-atlas"></i>&nbsp;Teams</a>
+          </li>
+          <li class="nav-item">
+            <a href="../FAQ/index.html" class="nav-link p-3" style="font-size: large;">
+              <i class="fas fa-question-circle"></i>&nbsp;FAQ</a>
+          </li>
+          <li class="nav-item">
+            <a href="../Minecraft_BTE_Tips/index.html" class="nav-link p-3" style="font-size: large;">
+              <i class="far fa-lightbulb"></i>&nbsp;Minecraft BTE Tips</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+  <br>
+  <br>
+  
+
+  <section class="index-news">
+    
+    <h1 style="text-align: center;color: black;">Tips for ASEAN BTE Builders</h1>
+    <div class="container px-5 py-24 mx-auto">
+      <div class="flex items-center lg:w-3/5 mx-auto border-b pb-10 mb-10 border-gray-200 sm:flex-row flex-col">
+        <div class="flex-grow sm:text-left text-center mt-6 sm:mt-0">
+          <h2 style="color: black;",>Worldedit</h2><article>
+          <p style="color: black;">You all know of the traditional way of building right? Well, we didnt use that. Instead, we implemented a plugin named Worldedit. Worldedit is a plugin that can place blocks and replace them at will. Its basically your bread and butter!</p>
```

Comment:
these boxes need better styling, gib them some padding and maybe better color.

Reply:
I guess the Teams pages cards would fit perfectly and with some kind of background, it would look even better I guess